.travis.yml

```diff
@@ -1,17 +1,33 @@
 language: bash
 env:
-    global:
-    - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+      global:
+      - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+
+dist: yakkety
+
+cache:
+  directories:
+    - $HOME/.cache/entities
 
 addons:
   apt:
     packages:
     - libxml2-utils
+    - libxml2-dev
+    - libxml2
 
 before_script:
+    - cat /etc/os-release
+    # - ls xml/*.xml
     - echo $TRAVIS_COMMIT
-    - ls xml/*.xml
     - xmllint --version
 
 script:
-    - xmllint --noout --noent xml/*.xml
+    - test -e $HOME/.cache/entities || mkdir -p $HOME/.cache/entities
+    - wget -P $HOME/.cache/entities http://docbook.org/xml/4.5/docbook-xml-4.5.zip
+    - unzip -d $HOME/.cache/entities/ $HOME/.cache/entities/docbook-xml-4.5.zip
+    - cp -vi $HOME/.cache/entities/*.mod xml/
+    - cp -avi $HOME/.cache/entities/ent xml/
+    # - wget -P xml/ http://www.oasis-open.org/docbook/xml/4.5/dbcentx.mod
+    - sed -i 's#dbcent PUBLIC .*>#dbcent SYSTEM "dbcentx.mod">#' xml/entity-decl.ent
```

Comment:
You can make this a bit safer by using `[^>]*`:

```
sed -i 's#dbcent PUBLIC [^>]*>#dbcent SYSTEM "dbcentx.mod">#' xml/entity-decl.ent
```

Otherwise, if you have another tag on the same line as this one, the second tag might be replaced too.